workloads/logging/deploy_logging_stack.sh

```diff
@@ -0,0 +1,102 @@
+}
+
+# Initialize kubeconfig to get access to the OpenShift cluster
+if [[ -n $KUBECONFIG_PATH ]]; then
+  export KUBECONFIG=$KUBECONFIG_PATH
+else
+  log "Looks like kubeconfig path is not set, assuming that it's in the default location - $HOME/.kube/config"
+fi
+
+# Check if oc client is installed
+log "Checking if oc client is installed"
+which oc &>/dev/null
+if [[ $? != 0 ]]; then
+  log "Looks like oc client is not installed, please install before continuing"
+  log "Exiting"
+  exit 1
+fi
+
+# Check if cluster exists and print the clusterversion under test
+oc get clusterversion
+if [[ $? -ne 0 ]]; then
+  log "Unable to connect to the cluster, please check if it's up and make sure the KUBECONFIG_PATH is set correctly"
+  exit 1
+fi
+
+# Vars and respective defaults
+export channel=${CHANNEL:=4.6}
+export es_node_count=${ES_NODE_COUNT:=3}
+export es_storage_class=${ES_STORAGE_CLASS:=gp2}
+export es_storage_size=${ES_STORAGE_SIZE:=100G}
+export es_memory_requests=${ES_MEMORY_REQUESTS:=8Gi}
+export es_proxy_memory_limits=${ES_PROXY_MEMORY_LIMITS:=256Mi}
+export es_proxy_memory_requests=${ES_PROXY_MEMORY_REQUESTS:=256Mi}
+export es_redundancy_policy=${ES_REDUNDANCY_POLICY:=SingleRedundancy}
+
+function install() {
+  # Create a Namespace for the Elasticsearch Operator
+  log "Creating a namespace for the Elasticsearch Operator"
+  oc create -f ./files/elasticsearch-operator-ns.yml
+
+  # Create a Namespace for the Cluster Logging Operator
+  log "Creating a namespace for the Cluster Logging Operator"
+  oc create -f ./files/cluster-logging-operator-ns.yml
+
+  # Create an Operator Group object YAML file for the Elasticsearch operator
+  log "Creating an Operator Group object YAML file for the Elasticsearch operator"
+  oc create -f ./files/elasticsearch-operator-group.yml
+
+  # Create a Subscription object YAML file to subscribe a Namespace to the Elasticsearch Operator
+  log "Creating a Subscription object YAML file to subscribe a Namespace to the Elasticsearch Operator"
+  envsubst < ./files/elasticsearch-operator-subscription.yml | oc create -f -
+
+  # Create an OperatorGroup object YAML file for the Cluster Logging Operator
+  log "Creating an OperatorGroup object YAML file for the Cluster Logging Operator"
+  oc create -f ./files/logging-operator-group.yml
+
+  # Create a subscription object YAML file to subscribe a Namespace to the Cluster Logging Operator
+  log "Creating a subscription object YAML file to subscribe a Namespace to the Cluster Logging Operator"
+  envsubst < ./files/logging-operator-subscription.yml | oc create -f -
+
+  # create a cluster logging instance
+  log "Creating a cluster logging instance"
+  envsubst < ./files/logging-instance.yml | oc create -f -
+}
+
+function cleanup() {
+  oc delete --wait=true project openshift-logging
+  oc delete --wait=true project openshift-operators-redhat
+  while [[ $( oc get projects | grep -w "openshift-logging\|openshift-operators-redhat") ]]; do
```

Comment:
Given that you're using the flag --wait=true I think that you don't need to double check if the project is still there.

Reply:
Appended the --ignore-not-found flag to avoid the check @rsevilla87. I am using v4.6.7 client and -wait=true seems to not wait for the finalizers, maybe it's a bug.

Reply:
yeah, it's possible, marking it as resolved then